sdf.md

```diff
@@ -209,15 +206,15 @@ Actions are used to model commands and methods which are invoked. Actions have p
 
 - Qualities of odmAction
 
-| Quality | Type | Optional | Description |
+| Quality | Type | Required | Description |
 |---|---|---|---|
-|id| integer | yes | internal unique identifier for the definition |
-|name|string|yes|human readable name|
-|description|string|yes|human readable description|
-|title|string|yes|human readable title to display|
-|optional| boolean|yes|defines whether this element is optional in an implementation|
-|include|array|yes|reference to definitions to be included|
-|type|object|yes|reference to a definition to be used as a template for a new definition|
+|id| integer | no | internal unique identifier for the definition |
```

Comment:
For consistency, Required here should probably be the same now as for others.

Reply:
fixed consistency: all id's are now required